node/src/types/block.rs

```diff
@@ -2154,7 +2153,7 @@ impl FinalitySignature {
     }
 
     /// Verifies whether the signature is correct.
-    pub fn verify(&self) -> crypto::Result<()> {
+    pub fn verify(&self) -> Result<(), crypto::Error> {
```

Comment:
That was defined in `node` and I didn't move the definition - but there is a reason for that, namely that now `crypto` has two error types: `Error` for errors that can happen in the `no_std` part, and `ErrorExt` for errors that can also happen when features using `std` are activated. So now we can't define a single `crypto::Result` type, as some methods will use `Result<T, Error>`, and some will use `Result<T, ErrorExt>`.
I could theoretically define a newtype for one of them, but it might cause some problems with shadowing the standard `Result` down the line.